org.eclipse.scout.rt.shared/src/main/java/org/eclipse/scout/rt/shared/services/common/code/IApiExposedCodeTypeContributor.java

```diff
@@ -18,12 +18,12 @@
  * Contributor for CodeTypes that should be exposed by the Scout REST api.
```

Comment:
Detail: Maybe we could move the part about the REST api to `IApiExposedCodeTypeDoProvider` as the resource now calls the provider instead of the contributor?

Reply:
Basically yes, but still the ApiExposedCodeTypeContributor only returns CodeTypes which should be available on the UI via REST. So the comment is still valid for the contributor as well. Therefore i would keep it like that. Ok?

Reply:
Fine by me 👍